--- a/plugins/dpdk.am
+++ b/plugins/dpdk.am
@@ -17,7 +17,7 @@ vppplugins_LTLIBRARIES += dpdk_plugin.la
 if ENABLE_DPDK_SHARED
 dpdk_plugin_la_LDFLAGS = $(AM_LDFLAGS) -ldpdk
 else
-dpdk_plugin_la_LDFLAGS = $(AM_LDFLAGS) -Wl,--whole-archive,-l:libdpdk.a,--no-whole-archive
+dpdk_plugin_la_LDFLAGS = -module -static -avoid-version -l:libdpdk.a
 endif
 if WITH_AESNI_MB_LIB
 dpdk_plugin_la_LDFLAGS += -Wl,--exclude-libs,libIPSec_MB.a,-l:libIPSec_MB.a
-- 
2.11.0

